scripts/jobs/cron_tasks.inc.http.30.nginx.php

```diff
@@ -224,7 +224,7 @@ public function createIpPort()
 					} else {
 						$_sslport = $this->checkAlternativeSslPort();
 						$mypath = 'https://' . Settings::Get('system.hostname') . $_sslport . '/';
-						$this->nginx_data[$vhost_filename] .= "\t" . 'if ($request_uri !~ ^/.well-known/acme-challenge/\w+$) {' . "\n";
+						$this->nginx_data[$vhost_filename] .= "\t" . 'if ($request_uri !~ ^/.well-known/acme-challenge/[\w-]+$) {' . "\n";
```

Comment:
To avoid mistakes/ambiguities on further extensions of the regex, with exception to the negation character ^ a dash should *always* be the first character within a character class.
--> `[-\w]+`